commands/deploy.go

```diff
@@ -128,6 +132,8 @@ func RunDeploy(
 		if parsedServices != nil {
 			services = *parsedServices
 		}
+	} else {
+		return fmt.Errorf("please provide a --name for your function as it will be deployed on FaaS")
```

Comment:
`FaaS` should be `OpenFaaS`

Can we re-write the message? It doesn't sound very natural.